docs/troubleshooting.md

```diff
@@ -0,0 +1,341 @@
+# Troubleshooting
+
+This is a guide on how to troubleshoot issues related to the Cluster API provider for vSphere (CAPV).
+
+* [Troubleshooting](#Troubleshooting)
```

Comment:
Recommend using https://github.com/ekalinin/github-markdown-toc in the future :)

Reply:
I do.

![ScreenCapture-2019 07 26-15 29 00](https://user-images.githubusercontent.com/101085/61979679-1b66a380-afba-11e9-82e6-7c3508682b43.png)

Reply:
(I just forgot to update the ToC)

Reply:
Rather I use https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one.